docs/modules/policies/pages/scope_permissions.adoc

```diff
@@ -0,0 +1,80 @@
+include::ROOT:partial$attributes.adoc[]
+
+= Scope Permissions
+
+include::ROOT:partial$version-check.adoc[]
+
+== Overview
+
+`scopePermissions` is a setting applied to all policies that impacts how rules are evaluated within a scope hierarchy. It defines whether policies in a given scope can **override** parent scope rules or whether they can only **restrict** the permissions granted by parent scopes.
+
+All policies within the same scope **must** use the same `scopePermissions` setting. If conflicting settings are detected within a shared scope, a build-time error will occur.
+
+There are two available settings:
+
+- `SCOPE_PERMISSIONS_OVERRIDE_PARENT`
+- `SCOPE_PERMISSIONS_REQUIRE_PARENTAL_CONSENT_FOR_ALLOWS`
+
+NOTE: By default, resource and principal policies use `SCOPE_PERMISSIONS_OVERRIDE_PARENT` unless explicitly set otherwise.
+
+IMPORTANT: Role policies require this field to be specified explicitly.
+
+== Scope Permissions: Override Parent
+
+`SCOPE_PERMISSIONS_OVERRIDE_PARENT` follows the pre-existing evaluation strategy. When this setting is applied, policies evaluate each action independently, progressing up the hierarchy for each action until a decision is reached.
+
+A child `OVERRIDE_PARENT` scope can **override** the behaviour of a parent if it has a matching rule for a given action. If no rule is found at the current scope for a specific action, evaluation continues up the hierarchy until a decision is made.
+
+=== Behaviour
+
+- Policies evaluate actions independently, resolving each action from the highest scope level up the chain until a decision is reached.
+- If an input matches a rule and optionally, its condition is met, we apply the effect of that rule.
+- If an input matches a rule but its condition is not met, or if no matching rule is found, evaluation continues up the hierarchy.
+
+== Scope Permissions: Require Parental Consent
+
+When a policy is configured with `scopePermissions: SCOPE_PERMISSIONS_REQUIRE_PARENTAL_CONSENT_FOR_ALLOWS`, it **inherits and restricts** the permissions of parent scopes. Policies at this level must define rules within the maximum set of permissions allowed by parent policies—they cannot introduce new permissions that exceed what a parent scope already permits.
```

Comment:
In this mode, an ALLOW rule that matches an action doesn't immediately generate an ALLOW decision. A parent policy higher up in the scope chain must also ALLOW that same action in order to produce a definitive decision.